src/fairseq2/models/w2vbert/builder.py

```diff
@@ -43,6 +43,7 @@ def _encoder_600m() -> Wav2Vec2EncoderConfig:
         layer_drop_p=0.0,
         norm_order=TransformerNormOrder.POST,
         depthwise_conv_kernel_size=31,
+        depthwise_conv_is_causal=False,
```

Comment:
Since the default is already `False` and this parameter is not relevant for w2v-BERT please remove this explicit argument pass.

Reply:
Nevermind. Ignore my comment. This is `Wav2Vec2EncoderConfig` so this parameter is required to be passed.